src/models/azure-key-vault-secret-status/index.ts

```diff
@@ -1,13 +1,23 @@
 import * as PropTypes from 'prop-types';
 
 export interface AzureKeyVaultSecretStatusModel {
-  replicaName?: string;
+  replicaName: string;
+  replicaCreated: Date;
+  jobName?: string;
+  jobCreated?: Date;
+  batchName?: string;
+  batchCreated?: Date;
   version?: string;
 }
 
 /* PropTypes validation map for AzureKeyVaultSecretStatusModel */
 export const AzureKeyVaultSecretStatusModelValidationMap: PropTypes.ValidationMap<AzureKeyVaultSecretStatusModel> =
   {
-    replicaName: PropTypes.string,
+    replicaName: PropTypes.string.isRequired,
+    replicaCreated: PropTypes.instanceOf(Date).isRequired,
```

Comment:
is `replicaCreated` always supplied by the API or is it actually optional?

Reply:
Always